Docker/entry.sh

```diff
@@ -3,6 +3,14 @@
 # Capture stack trace
 export RUST_BACKTRACE=1
 
+if [ -z "$VALIDATOR_ID" -a "$KUBERNETES_PORT" ]; then
+    export VALIDATOR_ID=${HOSTNAME##*-}
+    # assuming that WORKER_ID isn't also set.
+    # currently they match the validator it is assigned to.
+    export WORKER_ID=${VALIDATOR_ID}
```

Comment:
You could instead (and more safely, i.e. without clobbering a preexisting `WORKER_ID`) make `VALIDATOR_ID` the default value for `WORKER_ID`:
```bash
WORKER_ID=${WORKER_ID:=$VALIDATOR_ID}
```

Reply:
Thanks! good catch.